src/main/java/org/jenkinsci/plugins/gitserver/CSRFExclusionImpl.java

```diff
@@ -40,48 +41,50 @@
  */
 @Extension
 public class CSRFExclusionImpl extends CrumbExclusion {
+    private static final Set<String> ALLOWED_CONTENT_TYPES = Set.of(
+            "application/x-git-receive-pack-request",
+            "application/x-git-upload-pack-request"
```

Comment:
Turns out that this content type has been missing for who knows how long, so CSRF protection would have blocked any Git fetches to the Scriptler repository. Given that, I'm going to probably add functionality to enable/disable the Git repository (defaulting to disabled) and see what happens. I'm suspecting that no one uses it.

SSH access still works, but I feel like that would be less commonly used because you'd also have to enable the SSH server within Jenkins first.

Reply:
Just read the comments in #112 about HTTP access. Without knowing more about HTTP access, I'm removing this part from the commit and it can be dealt with separately.